### DISCUSSION TITLE
How to find "Allow Specified Actors to bypass required pull requests"?

### DISCUSSION THREAD

**Discussion by TimDMorris** [community/community#29771] [Enterprise] [answered]
[This page](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule) suggests I should be able to configure bypass of branch protection rules for specified actors.  It shows:

![allow bypass](https://user-images.githubusercontent.com/8735906/185065886-97de2436-07cc-4409-9d52-d0690f06ef0c.png)
 
However for [my repo](https://github.com/TimDMorris/ng-guitar) I cannot see this option.  If it makes any difference I pay for a Pro account, but unless I am missing something there doesn't seem to be any suggestion this is enterprise only etc.

I've searched stack overflow and I can see other people reporting this same behavior and asking the same question but I cannot see any answer anywhere can anyone please help?  Do you have this feature?  What type of account do you need to enable it? Or am I missing some configuration step?

https://stackoverflow.com/questions/72548343/where-is-the-allow-specified-actors-to-bypass-required-pull-requests-option

**Comment by HiBixby** [comment, marked as answer]
Hello!😊
I think it's because your repo doesn't belong to organization
I can see that option in my organization's repo even though it is private repository.
But I can not see that option in my personal repo which doesn't belong to organization even though it is public repository.
I hope this will help you 🙏

**Reply by HiBixby** [reply]
Oh It just popped into my head! did you add collaborators in your repo? that option (allow specified ...) seems to unable in personal repo, but you can add collaborator to your repository. It functions similarly.

Btw, you can create Github organization for free
you can see official docs from [here](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch)

**Reply by TimDMorris** [reply]
Hi  @HiBixby  thanks for the suggestions, yes I did add a collaborator to my repo but no my repo does not belong to an organization.  I will take a look at the link you shared and try setting up an organization and see if that helps.   

I will update if it works and then you can go collect the points on the stackoverflow question if you want them :)

**Reply by TimDMorris** [reply]
Hi @HiBixby just to confirm your suspicion was correct, I did not have this option because my repo was not part of an organization. I created an organization, moved the repo across to inside it and now I have the option.  Thanks for the help on this, much appreciated!

**Reply by HiBixby** [reply]
Congraturation! 🥳🥳🎉🎉
I am happy that this solution is helpful to you😊😊 Have a nice day!!

**Reply by Vixxd** [reply]
I had the exact same issue and have reached the same point in these steps, having reluctantly transferred my repo to an organisation to add this functionality.

However, now I see the following: 
![image](https://user-images.githubusercontent.com/5870958/217561406-f2b2294f-d392-4cde-8ee6-d001ca5e4703.png)

> Your protected branch rules won't be enforced on this private repository until you upgrade your organization account to a GitHub Team or Enterprise account.

But I pay for Pro on my Personal account - how can I avail of the "Allow Specified Actors to bypass required pull requests" functionality without duplicating payments on an organisation account and a personal account?

**Comment by AllanOricil** [comment]
This feature being available for enterprise or pro, and only for organizations, is a horrible, and unfair, business decision. It highlights you just wanted to get money from customers. Why dont you go ahead and move the entire branch protection rules to enterprise and pro accounts, then?? This would, at least, make your evil reasoning less obvious...
